notebooks/00_quick_start/run_notebook_on_aml.ipynb

```diff
@@ -0,0 +1,2380 @@
+{
```

Comment:
<p>I think it would be interesting to give an overview of how AzureML can help to execute notebooks remotely, maybe also explain what are the different compute targets available, etc</p>

 _Reply via <a href='https://app.reviewnb.com/Microsoft/Recommenders/pull/681/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
Added an overview